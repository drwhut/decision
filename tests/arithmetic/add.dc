Start~#1

> Adding 2 positive integers
Add(99, 1)~#2
Print(#1, #2)~#3

> Adding a positive and negative integer
Add(10, -10)~#4
Print(#3, #4)~#5

> Adding 2 negative integers
Add(-500, -500)~#6
Print(#5, #6)~#7

> Adding 2 positive floats
Add(9.56, 52.004)~#8
Print(#7, #8)~#9

> Adding a positive and negative float
Add(10.0, -12.5)~#10
Print(#9, #10)~#11

> Adding 2 negative floats
Add(-1.234, -5.678)~#12
Print(#11, #12)~#13

> Adding an integer and a float
Add(10, 5.5)~#14
Print(#13, #14)~#15