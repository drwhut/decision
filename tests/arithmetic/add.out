100
0
-1000
61.564
-2.5
-6.912
15.5
